test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs

```diff
@@ -84,12 +85,24 @@ protected async Task CosmosDBTest()
     public abstract class CosmosDBTestFixture : EndToEndTestFixture
     {
         protected CosmosDBTestFixture(string rootPath, string testId, string language) :
-            base(rootPath, testId, language, "Microsoft.Azure.WebJobs.Extensions.CosmosDB", "3.0.0-beta9*")
+            base(rootPath, testId, language)
         {
         }
 
         public DocumentClient DocumentClient { get; private set; }
 
+        protected override ExtensionPackageReference[] GetExtensionsToInstall()
```

Comment:
Hopefully we can get to the point where multiple extensions can be installed. This change will facilitate that. E.g. to enable the rest of the E2E samples for other extension types, we'll need to install multiple extensions.

Reply:
👍